site/content/events/2015-minneapolis/_sponsors.txt

```diff
@@ -23,7 +24,8 @@ end
 <h1><font color="silver">Silver Lake</font> sponsors</h1>
 
 <% @slsponsors = [
-{ :image => "ansible.png", :name => "Ansible", :ling => "http://www.ansible.com" }
+{ :image => "ansible.png", :name => "Anisble", :link => "https://www.ansible.com" },
+{ :iamge => "victorops.png", :name => "VictorOps", :ling => "https://www.victorops.com"}
```

Comment:
Typos:  `Anisble` should be `Ansible`, `iamge` should be `image`, `ling` should be `link`

You just want to see if I'm paying attention! :)